pages/index.js

```diff
@@ -1,27 +1,61 @@
 import Link from "next/link";
 import styled from "styled-components";
+import Clue from "../components/Clue";
+import { useEffect, useState } from "react";
+import useLocalStorageState from "use-local-storage-state";
+
+const StyledLayout = styled.div`
+  background-color: ghostwhite;
+  border: solid ghostwhite;
+  filter: ${(props) => (props.isOn ? "brightness(100%)" : "brightness(75%)")};
+  width: 375px;
+  height: 667px;
+`;
 
 const DoorButton = styled.button`
-  width: 100px;
+  width: 100spx;
   height: 150px;
   position: absolute;
-  transform: translate(0,-50%);
-  right: 20%;
-  top 50%;
-  font-size: 80px
+  transform: translate(0, -50%);
+  right: 0;
+  top: 50%;
+  font-size: 80px;
 `;
 
-const RoomDiv = styled.div`
-  display: relativ;
-  width: 300px;
+const LightButton = styled.button`
+  position: absolute;
+  transform: translate(0, -50%);
+  right: 100px;
+  top: 50%;
 `;
 
-export default function Room() {
+export default function Room({ randomColor, randomSymbol }) {
+  const [isOn, setIsOn] = useLocalStorageState("isOn", { defaultValue: true });
+  // Es wir nur beim zweiten Mal gerendert
+  const [isClient, setIsClient] = useState(false);
+  useEffect(() => {
```

Comment:
Why do you need the `useEffect` here? Couldn't you just set the default of `isClient` to `true`?

Reply:
I had the following problem:
React runs the code twice and creates a new color for my randomColor each time. This then gave me an error message that the color from the clint does not match the color from the server. 

But for now, my component is rendered on an onClick, and the useEffect is no longer needed. 
Thanks for the hint, I fixed it